src/main.rs

```diff
@@ -193,23 +39,23 @@ fn main() {
             "args"                  => println!("args = {:?}", tokens),
             "exit" | "q" | "quit"   => break,
             "controllers"           => {
-                let ref controllers = program.state.lock().unwrap().controllers;
+                let state = ozw.get_state();
```

Comment:
we need to bind this to a variable so that the borrow checker finds out we borrow this until the end of the block.